lib/usd/translators/shading/usdMaterialReader.cpp

```diff
@@ -84,13 +86,27 @@ bool PxrUsdTranslators_MaterialReader::Read(UsdMayaPrimReaderContext* context)
     context->RegisterNewMayaNode(prim.GetPath().GetString(), mayaObject);
 
     for (const UsdShadeInput& input : shaderSchema.GetInputs()) {
+        if (input.GetBaseName() == PxrMayaUsdPreviewSurfaceTokens->DisplacementAttrName) {
+            // We need a displacementShader:
+            std::string shaderName = prim.GetName().GetString();
+            shaderName += "_displacement";
+            UsdMayaTranslatorUtil::CreateShaderNode(
+                shaderName.c_str(),
+                "displacementShader",
+                UsdMayaShadingNodeType::Shader,
+                &status,
+                &_displacementShader);
+            // this displacement shader will get connected as we explore
+            // the displacement input of the UsdShadeMaterial.
```

Comment:
The material reader, which is responsible for backconverting from UsdPreviewSurface will now hold on to a displacementShader when there is an authored displacement input on the material.